rules/crowdstrike_rules/crowdstrike_real_time_response_session.py

```diff
@@ -0,0 +1,20 @@
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    return deep_get(event, "unknown_payload", "ExternalApiType") == "Event_RemoteResponseSessionStartEvent"
```

Comment:
deep_get requires a default if used for string comparison otherwise it will throw an error when comparing None to string